src/main/java/com/autotune/utils/AutotuneConstants.java

```diff
@@ -307,4 +307,17 @@ private ListDeploymentsInNamespace() { }
             }
         }
     }
+
+    public static class CORSConstants {
```

Comment:
javadoc here

Reply:
Thanks for the review @dinogun , Added the javadoc.